### PR TITLE
Show real rubric objects in teacher rubric view

### PR DESCRIPTION
**Pull request by bethanyaconnor**
Major changes in this PR:
- pass real rubric data down to the client and show real rubrics
- Styling of `RubricContainer` now that we see it with real content

Rubric on http://localhost-studio.code.org:3000/s/allthethings/lessons/48/levels/1?enableExperiments=ai-rubrics:

https://github.com/code-dot-org/code-dot-org/assets/46464143/1da65c1c-5b0b-4c87-84fc-749d88f4ff0b

Other changes:
- Modified the text components used for EvidenceLevels when feedback isn't available
- Fixed CSS specificity (I think) so that the text of `AiToken` is always green.

### CODE REVIEW

File: dashboard/app/models/rubric.rb [hunk 1]
```diff
@@ -17,6 +17,12 @@ class Rubric < ApplicationRecord
   belongs_to :level
   belongs_to :lesson
 
+  def summarize
```
Comment (bethanyaconnor) [inline dashboard/app/models/rubric.rb:20]: @kobryan0619 is there a possibility we've duplicated this function name?

File: apps/src/sites/studio/pages/levels/show.js [hunk 2]
```diff
@@ -55,6 +56,18 @@ function initPage() {
 
   const rubricFabMountPoint = document.getElementById('rubric-fab-mount-point');
   if (rubricFabMountPoint && experiments.isEnabled('ai-rubrics')) {
-    ReactDOM.render(<RubricFloatingActionButton />, rubricFabMountPoint);
+    const rubricData = getScriptData('rubric');
+    const reportingData = {
+      unitName: config.script_name,
+      courseName: config.course_name,
+      levelName: config.level_name,
```
Comment (davidsbailey) [inline apps/src/sites/studio/pages/levels/show.js:63]: update reportingDataShape to include courseName?

**Review by davidsbailey — APPROVED** [review]
this is exciting progress!

File: apps/src/sites/studio/pages/levels/show.js (hunk 2 above)
Comment (bethanyaconnor) [reply apps/src/sites/studio/pages/levels/show.js:63]: yes thank you!

File: dashboard/app/models/rubric.rb (hunk 1 above)
Comment (kobryan0619) [reply dashboard/app/models/rubric.rb:20]: Yes - I changed mine to `summarize_for_rubric_edit` so we should be good to go now though!

**Pull request merged**